### PR TITLE
Improve the email field validation in the home page

### PR DESCRIPTION
**Pull request by javisperez**
This PR uses a custom (better?) regexp pattern to validate emails in the newsletter subscription form.

### CODE REVIEW

**Review by amisevsk — COMMENTED** [review]
What, no +emails?

**Review by amisevsk — APPROVED** [review]

**Comment by javisperez** [comment]
That my friend @amisevsk was a bug :D nice catch. Updated it to suppot common email characters like `-`, `+`, `.` and `_`

File: docs/.vitepress/theme/components/Home.vue [hunk 1]
```diff
@@ -125,6 +125,7 @@ const subscribeToNewsletter = async () => {
           :disabled="isBusy"
           id="email"
           type="email"
+          pattern="^[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$"
```
Comment (gorkem) [inline docs/.vitepress/theme/components/Home.vue]: Can we improve it so that we allow for `name+test@mymail.com` kind of mails?

**Review by gorkem — CHANGES_REQUESTED** [review]

File: docs/.vitepress/theme/components/Home.vue (hunk 1 above)
Comment (javisperez) [reply docs/.vitepress/theme/components/Home.vue]: yeah, i just push a fix right before this comment :D it was my mistake, give it another try

Comment (javisperez) [reply docs/.vitepress/theme/components/Home.vue]: This is the commit https://github.com/jozu-ai/kitops/pull/539/commits/b550fd01d281bad30f5ca396153cc21547596be0

**Review by gorkem — APPROVED** [review]

**Pull request merged**